lib/common.rb

```diff
@@ -0,0 +1,10 @@
+module Common
+    # Make sure we are in dev mode unless explicitly disabled
+    def production_mode?
+        ENV['PRODUCTION_MODE'] == 'true' ? true : false
```

Comment:
Small nit, can you pls outdent these so the file has 2 spaces across the board, vs 4 spaces.